quint/src/flattening.ts

```diff
@@ -134,14 +148,15 @@ type DefAfterFlattening = (
 ) & WithOptionalDoc
 
 function isFlattened(def: QuintDef): def is DefAfterFlattening {
-  return def.kind !== 'instance' && def.kind !== 'import'
+  return def.kind !== 'instance' && def.kind !== 'import' && def.kind !== 'export'
```

Comment:
I am a bit confused by this name. So any definition different from an instance, `import`, and `export` will be considered flattened, even before flattening. Maybe we should rename the method to `isFlat`?